pkg/plugins/policies/meshtrafficpermission/api/v1alpha1/meshtrafficpermission.go

```diff
@@ -34,7 +34,8 @@ var ALLOW Action = "ALLOW"
 var DENY Action = "DENY"
 
 // ALLOW_WITH_SHADOW_DENY action lets the requests pass but emits logs as if
-//  requests are denied
+//
+//	requests are denied
```

Comment:
I think there is some problem with our dev tools not formatting the same way on different machines. Have you tried running `make dev/tools`?